pkgs/by-name/la/ladybird/package.nix

```diff
@@ -96,50 +96,58 @@ stdenv.mkDerivation (finalAttrs: {
     chmod +w build/Caches/AdobeICCProfiles
   '';
 
-  nativeBuildInputs = with qt6Packages; [
+  strictDeps = true;
+
+  nativeBuildInputs = [
     cmake
     ninja
     pkg-config
     python3
-    wrapQtAppsHook
-  ];
-
-  buildInputs = with qt6Packages; [
-    curl
-    ffmpeg
-    fontconfig
-    libavif
-    libGL
-    libjxl
-    libwebp
-    libxcrypt
-    openssl
-    qtbase
-    qtmultimedia
-    simdutf
-    (skia.overrideAttrs (prev: {
-      gnFlags = prev.gnFlags ++ [
-        # https://github.com/LadybirdBrowser/ladybird/commit/af3d46dc06829dad65309306be5ea6fbc6a587ec
-        # https://github.com/LadybirdBrowser/ladybird/commit/4d7b7178f9d50fff97101ea18277ebc9b60e2c7c
-        # Remove when/if this gets upstreamed in skia.
-        "extra_cflags+=[\"-DSKCMS_API=__attribute__((visibility(\\\"default\\\")))\"]"
-      ];
-    }))
-    woff2
-  ] ++ lib.optional stdenv.hostPlatform.isLinux [
-    libpulseaudio.dev
-    qtwayland
-  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
-    apple-sdk_14
+    qt6Packages.wrapQtAppsHook
   ];
 
-  cmakeFlags = [
-    # Disable network operations
-    "-DSERENITY_CACHE_DIR=Caches"
-    "-DENABLE_NETWORK_DOWNLOADS=OFF"
-  ] ++ lib.optionals stdenv.hostPlatform.isLinux [
-    "-DCMAKE_INSTALL_LIBEXECDIR=libexec"
-  ];
+  buildInputs =
+
```

Comment:
Why this extra line here?

Reply:
Good question...
Probably got there when I redid the diffs after moving to `pkgs/by-name`.

Will fix